File: client/iih_stub/iih.ci_stub/iih.ci.mrqc/autoqcdto/d/AutoQcDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.mrqc.autoqcdto.d
{
    /// <summary>
    /// AutoQcDTO 的摘要说明。
    /// </summary>
    public class AutoQcDTO : BaseDTO {

        public AutoQcDTO() {
 
        }

        /// <summary>
        /// 入院时间
        /// </summary>
		public DateTime? Dt_acpt {
            get { return getAttrVal<FDateTime>("Dt_acpt",null); }
            set { setAttrVal<FDateTime>("Dt_acpt", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Dt_acpt"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mrqc.autoqcdto.d.AutoQcDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mrqc/autoqcdto/d/AutoQcDTO.java
package iih.ci.mrqc.autoqcdto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 自动质控dto DTO数据 
 * 
 */
public class AutoQcDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 入院时间
	 * @return FDateTime
	 */
	public FDateTime getDt_acpt() {
		return ((FDateTime) getAttrVal("Dt_acpt"));
	}
	/**
	 * 入院时间
	 * @param Dt_acpt
	 */
	public void setDt_acpt(FDateTime Dt_acpt) {
		setAttrVal("Dt_acpt", Dt_acpt);
	}
}